packages/react-components/react-skeleton/stories/Skeleton/SkeletonBestPractices.md

```diff
@@ -2,4 +2,11 @@
 
 ### Do
 
+- Use `Skeleton` to help ease a UI transition when we know the service will potentially take a longer amount of time to retrieve the data.
+- Provide widths for each of the `Skeleton` elements you used to build a skeleton layout looking as close as possible to real content it is replacing.
+- Use `Skeleton` if you know the UI loading time is longer than 1 second.
+
 ### Don't
+
+- Build Skeleton UI should with a lot of details. Circles and rectangles are really as detailed as you want to get. Adding more detail will result in confusion once the UI loads.
```

Comment:
Typo: extra "should". This might also need to be edited to make it a little less informal.
```suggestion
- Build Skeleton UI with a lot of details. Circles and rectangles are really as detailed as you want to get. Adding more detail will result in confusion once the UI loads.
```